pages/requests/index.js

```diff
@@ -6,54 +6,52 @@ import {
   LinkedButton,
   Loading,
   RequestList,
-  Title,
 } from '@scientist-softserv/webstore-component-library'
 import {
   configureErrors,
   dark,
   useDefaultWare,
-  useAllRequests 
+  useAllRequests
 } from '../../utils'
 
-// TODO(alishaevn): authenticate the user
-const Requests = ({ ...props }) => {
+const Requests = () => {
   const router = useRouter()
   const { data: session } = useSession()
-  const { requests, isLoadingAllRequests, isAllRequestsError } = useAllRequests()
-  const { user, userError, userLoading } = props
-  const { defaultWareID, isLoadingDefaultWare, isDefaultWareError } = useDefaultWare()
-  const isError =  isAllRequestsError || isDefaultWareError || userError
-  const isLoading = isLoadingAllRequests || userLoading || isLoadingDefaultWare
+  const { requests, isLoadingAllRequests, isAllRequestsError } = useAllRequests(session?.accessToken)
+  const { defaultWareID, isLoadingDefaultWare, isDefaultWareError } = useDefaultWare(session?.accessToken)
+  const isError = isAllRequestsError || isDefaultWareError
+  const isLoading = isLoadingAllRequests || isLoadingDefaultWare
+
+  if (!session) {
+    return (
+      <Error
```

Comment:
I considered using `configureErrors`...but we don't actually have an error. this is a path that's blocked by auth. so I went this route instead.

however...my more alert mind agrees with you. I would just not use the `Error` component at all though instead of trying to finagle the component to do something it wasn't intended to.

the alert will remain outside the body return.

Reply:
actually, I'll refactor the Error component to handle other alert types. 
re: https://github.com/scientist-softserv/webstore-component-library/issues/136